.gitignore

```
@@ -1,5 +1,4 @@
 doc
-shard.lock
```

```suggestion
shard.lock
```

no reason to have a lock file in a lib